packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -96,7 +96,10 @@ export class IndexedDbPersistence implements Persistence {
   static MAIN_DATABASE = 'main';
 
   private simpleDb: SimpleDb;
-  private started: boolean;
+  private hasStarted: boolean;
```

Comment:
The general convention for exposing properties that are only readonly via the public API is to use an underscore. So this would be:

```
private _started: boolean;
get started(): boolean {
    return this._started;
}
```

Please also move the getter to where the other getters/methods are defined. You could also initialize _started with 'false'.

Reply:
Done.